src/components/app/data/hooks/useEnterpriseLearner.js

```diff
@@ -12,8 +12,41 @@ import { queryEnterpriseLearner } from '../queries';
 export default function useEnterpriseLearner(queryOptions = {}) {
   const { authenticatedUser } = useContext(AppContext);
   const { enterpriseSlug } = useParams();
-  return useQuery({
-    ...queryEnterpriseLearner(authenticatedUser.username, enterpriseSlug),
-    ...queryOptions,
+  const { select, ...queryOptionsRest } = queryOptions;
+
+  return useBFF({
+    bffQueryOptions: {
+      ...queryOptionsRest,
+      select: (data) => {
+        const transformedData = {
+          enterpriseCustomer: data.enterpriseCustomer,
+          allLinkedEnterpriseCustomerUsers: data.allLinkedEnterpriseCustomerUsers,
+          enterpriseFeatures: data.enterpriseFeatures,
+        };
+        // When custom `select` function is provided in `queryOptions`, call it with original and transformed data.
+        if (select) {
+          return select({
+            original: data,
+            transformed: transformedData,
+          });
+        }
+
+        // Otherwise, return the transformed data.
+        return transformedData;
+      },
+    },
+    fallbackQueryConfig: {
+      ...queryEnterpriseLearner(authenticatedUser.username, enterpriseSlug),
+      ...queryOptions,
+      select: (data) => {
+        if (select) {
+          return select({
```

Comment:
nit: could be worth a comment to inform why we pass `data` as both `original` and `transformed`, e.g.

```js
// To maintain parity with BFF-enabled routes in the function signature passed to the custom `select` 
// function, the legacy `queryEnterpriseLearner` also passes its `data` to the custom `select` function
// as both the `original` and `transformed` properties, since no data transformations occur here.
```